cmd/puller/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"flag"
+	"github.com/kubeflow/kfserving/pkg/puller"
+	"log"
+)
+
+var (
+	modelDir = flag.String("model_dir", "/tmp","directory for multi-model config files")
+)
+
+func main() {
+	flag.Parse()
+	puller.OnConfigChange(func(e puller.EventWrapper) {
+		log.Println("Send a request to:", e.LoadState)
+		log.Println("for model", e.ModelName)
+	})
+	puller.WatchConfig(*modelDir)
```

Comment:
Should we use "informers" here? https://firehydrant.io/blog/stay-informed-with-kubernetes-informers/

Reply:
@ellistarn We'd like to not depend on k8s for model puller, likewise for model server.

Reply:
Let me know when there is a verdict on this.

Reply:
I tend to agree with @yuzisun on this